api/grpcserver/grpcserver_test.go

```diff
@@ -54,6 +51,10 @@ import (
 	"github.com/spacemeshos/go-spacemesh/sql/statesql"
 	"github.com/spacemeshos/go-spacemesh/system"
 	"github.com/spacemeshos/go-spacemesh/txs"
+	"github.com/spacemeshos/go-spacemesh/vm"
+	vmsdk "github.com/spacemeshos/go-spacemesh/vm"
```

Comment:
The `vm` is imported twice
```suggestion
	"github.com/spacemeshos/go-spacemesh/vm"
```